nat-lab/bin/kill_process_by_natlab_id

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
This script introduces even more hidden dependencies: bash, grep, ps.
Are they installed on the nat-lab containers? I would recommend at least using POSIX sh instead of bash

Reply:
There are a few things here:
 1. All of the our scripts use bash, I think we should keep this consistent
 2. ps and grep are quite basic tools which are installed on the Ubuntu based docker containers and both of them are already used is our scripts
 3. I think we should strongly prefer extending docker images over adding complexity to the scripts